src/styles/main.scss

```diff
@@ -1,3 +1,170 @@
-body {
+// #region page
+.page {
+  font-family: Manrope, sans-serif;
+  color: #1B2129;
+  font-size: 16px;
+  line-height: 140%;
+
+  scroll-behavior: smooth;
+}
+
+.page__body {
+  margin: 0;
+}
+
+.page__menu {
+  position: fixed;
+  top: 0;
+  right: 0;
+  left: 0;
+
+  opacity: 0;
+  pointer-events: none;
+}
+
+.page__menu:target {
+  opacity: 1;
+  pointer-events: all;
+}
```

Comment:
It's better to use nesting as you have SCSS file which alows it. Try to implement it in all the possible places.